app/celery/letters_pdf_tasks.py

```diff
@@ -219,3 +224,29 @@ def update_letter_pdf_status(reference, status):
             'status': status,
             'updated_at': datetime.utcnow()
         })
+
+
+def replay_letters_in_error(filename=None):
+    # This method can be used to replay letters that end up in the ERROR directory.
+    # We had an incident where clamAV was not processing the virus scan.
+    if filename:
+        move_error_pdf_to_scan_bucket(filename)
+        # call task to add the filename to anti virus queue
+        current_app.logger.info("Calling scan_file for: {}".format(filename))
+        notify_celery.send_task(
+            name=TaskNames.SCAN_FILE,
+            kwargs={'filename': filename},
+            queue=QueueNames.ANTIVIRUS,
+        )
+    else:
+        error_files = get_file_names_from_error_bucket()
+        for item in error_files:
+            moved_file_name = item.key.split('/')[1]
+            current_app.logger.info("Calling scan_file for: {}".format(moved_file_name))
```

Comment:
This and the next 7 lines could be replaced with `replay_letters_in_error(moved_file_name)`